### PR TITLE
Add info about remotely-hosted code to 'What's new'.

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by jpmedley** [comment]
@oliverdunk, Sebastian didn't get to this before he left. Can you please take a look.

File: site/en/docs/extensions/whatsnew/index.md
Comment (oliverdunk) [inline site/en/docs/extensions/whatsnew/index.md:22]: I'd prefer to get rid of the "has been part of our guidance since the beginning", since we published the migration guide fairly late and it seems like an unnecessary can of worms to open (also fixed some typos and a missing link):

```suggestion
A requirement for Manifest V3 is that extensions may no longer use remotely-hosted code. We want to improve the guidance around this issue, and have made [additions to the migration guide](docs/extensions/migrating/improve-security/) to provide more information, describing what's still possible in Manifest V3 and more information on strategies for upgrading.
```

**Review by oliverdunk — CHANGES_REQUESTED** [review]
Thanks for this! I requested a couple of changes and also I assume we need to wait for the PR to merge: https://github.com/GoogleChrome/developer.chrome.com/pull/7000

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:24]:
```suggestion
There is a related addition to [Troubleshooting Chrome Web Store violations](/docs/webstore/troubleshooting/). A new section describes common reasons that extensions with remotely-hosted code are rejected.
```

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [reply site/en/docs/extensions/whatsnew/index.md]: I only did half the edit I intended here. Sorry about that. Please see the new line 24.

**Comment by jpmedley** [comment]
PTAL

**Review by oliverdunk — APPROVED** [review]
LGTM, although adding Don't merge label until Sebastian's PR is landed.

**Pull request closed without merging**
